setup.py

```diff
@@ -1,84 +1,41 @@
-# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
```

Comment:
Can you preserve the license after the shebang line.

Also retain any relevant docs on env var usage.